Sources/Basic/FileInfo.swift

```diff
@@ -8,30 +8,23 @@
  See http://swift.org/CONTRIBUTORS.txt for Swift project authors
  */
 
-import SPMLibc
 import Foundation
 
 /// File system information for a particular file.
 public struct FileInfo: Equatable, Codable {
```

Comment:
`FileInfo` is stored in the manifest db so we need to bump the db schema: https://github.com/apple/swift-package-manager/blob/master/Sources/SPMLLBuild/llbuild.swift#L98